src/docs/product/discover-queries/index.mdx

```diff
@@ -21,7 +21,9 @@ This feature is available only if your organization is on either a Business or T
 
 ![Full view of the Discover Homepage with query cards and button to build new queries.](./discover-homepage.png)
 
-The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> page offers views of all your saved and pre-built queries so you can quickly see areas that need immediate attention. For example, if a query on errors shows a spike has occurred in the past 24 hours, you might want to investigate that first. Each query is saved as a card that displays a summarized view of the following data:
+The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage will open the query builder where you can immediately begin building a new query. By default, this will show the pre-built "All Events" query. Columns in the homepage can be [edited and saved, or saved from other queries](#set-as-default). To navigate back to the [query management page](#saved-queries), where pre-built and saved queries can be found, click the "Saved Queries" button.
```

Comment:
A couple things:

- I don't understand what you mean by "saved from other queries" and the section on default homepages doesn't really explain this so I've edited it to mirror the wording we used in that section
- I understand putting the link in the words "query management page" but it's confusing to link it to a heading called Saved Queries so I've moved the link.

```suggestion
The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin building a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set it as your default](#set-as-default). To navigate back to the query management page(#saved-queries), where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```

Reply:
Let's do it!